docs/upgrade-guide.md

```diff
@@ -14,7 +14,7 @@ Here's a short list of thing that need to be updated:
     directly. The
     [`package.json`](https://github.com/dhis2/aggregate-data-entry-app/blob/master/package.json)
     of the Aggregate Data Entry app is a good example.
--   Cypress needs to be upgraded to version ^12.7.0
+-   Cypress needs to be upgraded to version ^13.10.0
```

Comment:
```suggestion
-   Upgrade cypress to the lastest version (must be `>=12.7.0`)
```

Reply:
Future proofing this a little, as this will probably work with future major releases, too.